src/models/tiles/tile-content-info.ts

```diff
@@ -17,6 +19,12 @@ type TileModelSnapshotPreProcessor = (tile: any) => any
 type TileContentSnapshotPostProcessor =
       (content: any, idMap: Record<string, string>, asTemplate?: boolean) => any;
 
+type TileContentNewSharedModelIdUpdater = (
+  content: any,
```

Comment:
A better type for this would be `unknown`.  This will force the `updateContentWithNewSharedModelIds` to make sure they are being passed the right thing.

But this isn't critical for completing this PR.